lizmap/modules/admin/install/upgrade_newAclRights.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+class lizmapModuleUpgrader_newSubjectsAndRights extends jInstallerModule
+{
+    public $targetVersions = array(
+        '3.7.0',
+        '3.6.1',
+        '3.6.0',
```

Comment:
Ok, but what about the date ? Since I do not know the date of the next Lizmap 3.6.1 release. Should I use today ?

```php
public $date = '2022-01-06';
```

Reply:
yes, today

Reply:
But probably, to be able to use this new feature right now, we should release a beta, so you would indicate 3.6.1-beta.1 for example...